### PR TITLE
remove boost version dependent code

### PR DESCRIPTION
**Pull request by winsvega**
this might be not needed for actual testing. but it makes testeth to crash each time boost lib is updated.

### CODE REVIEW

**Comment by LefterisJP** [comment]
It's really good to remove this.

But any idea why the ubuntu tests are failing?

```
Stacktrace

[Exception] - critical check uncleBlockQueue.blockStatus(block2.blockHeader().hash()) == QueueStatus::Bad failed
 == [File] - /var/jenkins_home/sharedspace/libethereum/label/master/libethereum/test/libethereum/BlockChain.cpp
 == [Line] -180
```

[Link](http://52.28.164.97/job/project-test/label=master/1254/testReport/junit/Master.Test.Suite.BlockChainSuite/_var_jenkins_home_sharedspace_libethereum_label_master_libethereum_test_TestHelper/sync/) to failure

**Comment by chriseth** [comment]
@LefterisJP that failure has happened before and is unrelated. We will focus on testing this and the following week and hopefully also fix that problem.

File: test/boostTest.cpp [hunk 1]
```diff
@@ -110,21 +110,6 @@ int main( int argc, char* argv[] )
 	try
 	{
 		framework::init(fake_init_func, argc, argv);
-#if BOOST_VERSION >= 106000
```
Comment (chriseth) [inline test/boostTest.cpp:113]: What was the purpose of this code if it can be simply removed?

Comment (LefterisJP) [inline test/boostTest.cpp:113]: Ask @winsvega , the very first usage of boost internal code is by him. I just had to go in and add a new condition each time boost upgraded its minor version.

Comment (winsvega) [inline test/boostTest.cpp:113]: it is not my code neither. It was here before and it is just appears that it might not even be needed.

**Pull request merged**